watson-developer-cloud/discovery-components
# chore: upgrade react-scripts

#### What do these changes do/fix?

- Upgrade to `react-scripts` 5.0.1. Upgrade related dependencies (e.g. `eslint`)
- Replace `node-sass` (deprecated) with `sass`/`sass-embedded`
- Upgrade to Storybook v6.x (needed since that supports Webpack v5, which is used by latest `react-scripts`)
- Upgrade to Cypress v10. Update tests to match changes.

Fixes #391

#### How do you test/verify these changes?

- Run example app
- Run storybook
- Link into Tooling and verify

#### Have you documented your changes (if necessary)?

- Updated main README.md to replace `node-sass` with `sass`

#### Are there any breaking changes included in this pull request?
